index.js

```diff
@@ -135,6 +142,10 @@ class ClinicBubbleprof extends events.EventEmitter {
     })
     b.add(scriptPath)
     const scriptFile = b.bundle()
+
+    if (!this.debug) {
+      scriptFile.pipe(minifyStream({ sourceMap: false, mangle: false }))
```

Comment:
You'll need to do `scriptFile = scriptFile.pipe(minifyStream({ sourceMap: false, mangle: false }))`. Otherwise `minifyStream`  would not go anywhere:

```js

varvar minifyStream  mini = require('minify-stream')
 
fs.createReadStream('app.js')
  .pipe(minifyStream())
  .pipe(fs.createWriteStream('app.min.js'))
```

Also, I would `pump` them.

Reply:
Oops